src/GraphQL/Attributes/ImplementsAttribute.cs

```diff
@@ -0,0 +1,45 @@
+using GraphQL.Types;
+
+namespace GraphQL
+{
+    /// <summary>
+    /// Marks the type as implementing a specified interface.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class)]
+    public class ImplementsAttribute : GraphQLAttribute
```

Comment:
I think you forgot to add `AllowMultiple = true` that is `false` by default.

Reply:
Done + added test